### ISSUE TITLE
CI recommendations

### ISSUE THREAD

**Issue by frewsxcv** [georust/meta#14]
Prefer GitHub Actions over Travis?

**Comment by michaelkirk** [comment]
I don't want to be *overly* prescriptive, especially about any details that are likely to change, but a certain amount of management uniformity could ease the maintenance burden, especially as the number of repos we have grows.

How about:

1. some level of CI
2. bors integration which triggers CI and merges upon success

Anything to add? Anything to take away?

**Comment by urschrei** [comment]
I think those are the minimum sensible requirements. I'd be comfortable with preferring (not prescribing) GitHub Actions as of 2022 with the understanding that this could change (could revisit once a year to update to "as of 2023" if need be). WDYT?

**Comment by michaelkirk** [comment]
Yes, good idea. I'm happy to recommend specifically using github actions for now.
